www/docs/installation.mdx

```diff
@@ -0,0 +1,33 @@
+---
+id: installation
+title: Installation
+---
+
+Effection is designed to for gradual adoption. You can use as a little or as much Effection as you need.
+You can use Effection as a replacement for an async/await within your existing application, manage all of
+async operations in your application or anything in-between.
+
+Effection can be used in any JavaScript environment including Deno, Node and browsers. Effection packages are side-effect free and do not pollute your environment in any way, making it compatible with most JavaScript runtimes.
+If you encounter obstacles integrating with your environment, please create a [GitHub Issue in the Effection repository](https://github.com/thefrontside/effection/issues/new).
+
+Effection comes with [first-class TypeScript support](https://arethetypeswrong.github.io/?p=starfx%400.0.32), [is tree-shackable, has no dependencies and is small](https://bundlephobia.com/package/effection@3.0.0-beta.1).
+
+## NPM/Yarn
+
+Effection packages are available via NPM and Yarn. The package includes ESM, CommonJS and TypeScript types.
+
+```bash
+// install with npm
+npm install effection@v3
+
+// install with yarn
+yarn add effection@v3
```

Comment:
```suggestion
// install with npm
npm install effection@next

// install with yarn
yarn add effection@next
```

Reply:
Turns out `v3` is not a valid npm dist tag

Reply:
Yeah, I noticed that too